src/main/java/org/broadinstitute/consent/http/models/LibraryCard.java

```diff
@@ -160,4 +167,26 @@ public void removeDaa(Integer daaId) {
       this.daaIds.remove(daaId);
     }
   }
+
+  public void addDaaObject(DataAccessAgreement daa) {
+    if (this.daas == null) {
+      this.daas = new ArrayList<>();
+    }
+    if (this.daas
+        .stream()
+        .noneMatch(d -> d.equals(daa))) {
+      this.daas.add(daa);
+    }
+  }
+
+  public void removeDaaObject(DataAccessAgreement daa) {
+    if (this.daas == null) {
+      return;
+    }
+    if (this.daas
+        .stream()
+        .anyMatch(d -> d.equals(daas))) {
+      this.daas.remove(daas);
+    }
+  }
```

Comment:
Two things
1. this method is unused - I suggest removing it - if we need it for some other feature, we can always add it.
2. the `daa` in this method is unused - I think what you want in the second if condition is
```
    if (this.daas
        .stream()
        .anyMatch(d -> d.equals(daa))) {
      this.daas.remove(daa);
    }
```